### DISCUSSION TITLE
Hexagon preset dont work

### DISCUSSION THREAD

**Discussion by capoia** [matteobruni/tsparticles#4954] [Q&A] [answered]
Good afternoon, I'm trying to replicate the hexagon demo on the site, but when I use the .json that is available, the effect is different, it doesn't form anything.

I realized that inside the builder it works perfectly, but when exporting to codepen it stops working correctly.

https://codepen.io/capoia/pen/ZEqevKR

**Comment by matteobruni** [comment, marked as answer]
https://github.com/matteobruni/tsparticles/issues/3102
